101-vm-sql-autobackup-update/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Setup or update SQL Server Auto Backup setting with an Azure Storage Account",
```

Comment:
In the display name and description it be good to call out there's an existing vm involved as well